lib/tasks/deny_missed_requests.rake

```diff
@@ -2,12 +2,16 @@ desc 'Mark missed requests as denied'
 task deny_missed_requests: :environment do
   # get all requests that began yesterday and weren't approved / denied /
   # acted upon
+  # also sends an email to the user
```

Comment:
Is there any danger that this would keep sending this e-mail every night for a given missed request until the reservation was actually destroyed? I'm not seeing anything in the code / scope to limit this to running once per reservation. We might have to update the `missed_request` scope to exclude those that are flagged `expired`.